package.json

```diff
@@ -25,7 +25,10 @@
     "pranaygp.vscode-css-peek",
     "patbenatar.advanced-new-file",
     "akamud.vscode-theme-onedark",
-    "akamud.vscode-theme-onelight"
+    "akamud.vscode-theme-onelight",
+    "formulahendry.auto-rename-tag",
```

Comment:
Here We just add plugins that exist in both stores [Microsoft](https://marketplace.visualstudio.com) and [openVSX](https://open-vsx.org)

Reply:
aaaaah.
This was kinda weeird to me at the start.
Also i couldnt really test it since the test itself didnt work for me *apparently*